src/components/recipe-form.js

```diff
@@ -283,10 +283,16 @@ const RecipeForm = ({
                         arrayHelpers.push({ name: '', amount: '', unit: '' })
```

Comment:
The `UnitDropdown` is either transparent or inheriting the background color from the container. I feel like it should be white like the other inputs

Reply:
Fixed this by removing the background color, it still looks weird. Just can't seem to find a good grayish color for it so white for now I suppose 🤷‍♂